ConsentViewController/Classes/SPError.swift

```diff
@@ -135,7 +136,7 @@ import Foundation
 }
 
 @objcMembers public class RenderingAppError: SPError {
-    override public var spCode: String { renderingAppErrorCode ?? "sp_metric_rendering_app_error" }
+    override public var spCode: String { renderingAppErrorCode ?? "sp_metric_webview_creation_error" }
```

Comment:
Warning! This might be an unnecessary change. We have to discuss it.

Reply:
Gotta revert it back, one moment

Reply:
I believe that's not correct. `RenderingAppError` is used for the case when the rendering app emits a `error` event and we capture that via the JSReceiver.

Reply:
@andresilveirah this change was reverted back so the code remains untouched. My bad.